pkg/certificates/certs.go

```diff
@@ -62,17 +61,15 @@ func createCACertTemplate(expirationInterval time.Duration) (*x509.Certificate,
 	}
 	// Make it into a CA cert and change it so we can use it to sign certs
 	rootCert.IsCA = true
-	rootCert.KeyUsage = x509.KeyUsageCertSign | x509.KeyUsageDigitalSignature
```

Comment:
+1 
From the RFC :

>   The digitalSignature bit is asserted when the subject public key
      is used for verifying digital signatures, other than signatures on
      certificates (bit 5) and CRLs (bit 6), such as those used in an
      entity authentication service, a data origin authentication
      service, and/or an integrity service.

Wondering why was that there.